@here/harp-mapview/lib/Utils.ts

```diff
@@ -32,6 +34,7 @@ const yawQuaternion = new THREE.Quaternion();
 const pitchQuaternion = new THREE.Quaternion();
 const tmpQuaternion = new THREE.Quaternion();
 const tmpMatrix = new THREE.Matrix4();
+const tmpVectors = new Array<THREE.Vector3>(3).fill(new THREE.Vector3());
```

Comment:
It's wrong, you're filled whole array with one vector instance

Reply:
Hmm... looks so, but why it's still working, that is kind of mystery... especially that I am using two array elements at the same time :).
Will fix it though...